docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -93,7 +98,10 @@ Example of using lists to set shapes, types and layout for multiple inputs:
 .. code-block:: py
    :force:
 
-   ov_model = convert_model(model, input=[("input1", [1,3,100,100], np.float32), ("input2", [1,3,100,100], np.float32)], layout=[Layout("NCHW"), LayoutMap("NCHW", "NHWC")])
+   import openvino as ov
+   from openvino.tools.mo import LayoutMap
+
+   ov_model = ov.convert_model(model, input=[("input1", [1,3,100,100], np.float32), ("input2", [1,3,100,100], np.float32)], layout=[ov.Layout("NCHW"), LayoutMap("NCHW", "NHWC")])
```

Comment:
What about `from openvino.tools.mo import LayoutMap` and it's usage later. @slyalin `ov.convert_model` is ovc-based, what happens when mo-based arguments are passed around?

Reply:
`ovc` doesn't have `layout` parameter. This snippet is no more relevant to the top-level API we have. Please avoid modifying snippets in MO part of the docs we are going to keep it as is including `openvino.tools.mo.convert_model`.

Reply:
I have reverted them in the https://github.com/openvinotoolkit/openvino/pull/19084/commits/24d1321aa23402e051f7925957e6eedaacd7d9ff commit.